apps/web/utils/string.tsx

```diff
@@ -33,12 +34,14 @@ export const parseStringToComponents = (html: string) => {
 
             if (DefinedTermComponent) {
               return (
-                <DefinedTermComponent
-                  {...(props as unknown as DefinedTermProps)}
-                  key={domNode.attribs.key}
-                >
-                  {domToReact(domNode.children as DOMNode[], options)}
-                </DefinedTermComponent>
+                  <Tooltip tooltipContent="Default Tooltip" triggerContent={
```

Comment:
Ideally I would think this wrapper happen directly in the DefineTerm component. That way if we ever need to directly use the DefinedTerm component outside of parsing an HTML string we still get the tooltip with it. Was that not possible based on some setup here? Thoughts?